content/en/docs/tutorials/acme/ingress.md

```diff
@@ -222,30 +221,16 @@ application which makes an excellent back-end for examples.
 The quick-start example uses three manifests for the sample. The first two are a
 sample deployment and an associated service:
 
-TODO (`@joshvanl`): move this link to the new location
-- deployment manifest:
-  [`deployment.yaml`](https://raw.githubusercontent.com/jetstack/cert-manager/release-0.12/docs/tutorials/acme/quick-start/example/deployment.yaml)
-
-TODO (`@joshvanl`): include link contents
-```yaml
-example/deployment.yaml
-```
-
-TODO (`@joshvanl`): move this link to the new location
-- service manifest:
-  [`service.yaml`](https://raw.githubusercontent.com/jetstack/cert-manager/release-0.12/docs/tutorials/acme/quick-start/example/service.yaml)
+{{% include file="/content/en/docs/tutorials/acme/example/deployment.yaml" language="yaml" %}}
 
-TODO (`@joshvanl`): include link contents
-```yaml
-example/service.yaml
-```
+{{% include file="/content/en/docs/tutorials/acme/example/service.yaml" language="yaml" %}}
```

Comment:
We need to use relative paths here as this file is in `content/en/docs`, meaning it is subject to versioning